SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Utils/RPCContainer/RPCContainerTest.cpp

```diff
@@ -215,8 +215,15 @@ RPCCONTAINER_TEST(GIVEN_a_container_with_one_value_WHEN_processing_after_RPCQueu
 
 	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 	check(SpatialGDKSettings != nullptr);
-	FPlatformProcess::Sleep(SpatialGDKSettings->RPCQueueWarningDefaultTimeout);
+	FPlatformProcess::Sleep(SpatialGDKSettings->RPCQueueWarningDefaultTimeoutSeconds);
 	RPCs.ProcessRPCs();
+	constexpr float ErrorRetryWaitTimeSeconds = 0.1f;
+	int NrRetries = 0;
```

Comment:
```suggestion
	int32 NrRetries = 0;
```
prefer sized integers